bin/gltf-pipeline.js

```diff
@@ -15,22 +15,24 @@ var readGltf = require('../lib/readGltf');
 if (process.argv.length < 3 || defined(argv.h) || defined(argv.help)) {
     var help =
         'Usage: node ' + path.basename(__filename) + ' [path-to.gltf or path-to.bgltf] [OPTIONS]\n' +
-        '  -i, input=PATH Read unoptimized glTF from the specified file.\n ' +
-        '  -o, output=PATH write optimized glTF to the specified file.\n' +
-        '  -b, write binary glTF file.\n' +
-        '  -s, writes out separate geometry/animation data files, shader files and textures instead of embedding them in the glTF file.\n  ';
+        '  -i --input, input=PATH Read unoptimized glTF from the specified file.\n' +
+        '  -o --output, output=PATH write optimized glTF to the specified file.\n' +
+        '  -b --binary, write binary glTF file.\n' +
+        '  -s --separate, writes out separate geometry/animation data files, shader files and textures instead of embedding them in the glTF file.\n' +
+        '  -t --separateImage, write out separate textures, but embed geometry/animation data files, and shader files.\n';
     process.stdout.write(help);
     return;
 }
 
-var gltfPath = defaultValue(argv._[0], argv.i);
+var gltfPath = defaultValue(argv._[0], argv.i || argv.input);
 var fileExtension = path.extname(gltfPath);
 var fileName = path.basename(gltfPath, fileExtension);
 var filePath = path.dirname(gltfPath);
 
-var outputPath = defaultValue(argv._[1], argv.o);
-var isSeparate = defaultValue(argv.s, false);
-var isBinary = defaultValue(argv.b, false);
+var outputPath = defaultValue(argv._[1], argv.o || argv.output);
```

Comment:
To follow the coding guide, just do `defaultValue(argv._[1], defaultValue(argv.o, argv.output))` for these instead of `||`.